### PR TITLE
feat: generate SDKs for Looker 23.8

### PR DESCRIPTION
**Pull request by drstrangelooker**
Release-As: 23.8.0

### CODE REVIEW

**Comment by drstrangelooker** [comment]
@jkaster @jeremytchang typescript tests are failing...

```
Found duplicated entries. Run yarn-deduplicate to deduplicate them.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 1.
```

**Comment by jkaster** [comment]
> @jkaster @jeremytchang typescript tests are failing...
> 
> ```
> Found duplicated entries. Run yarn-deduplicate to deduplicate them.
> error Command failed with exit code 1.
> info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
> Error: Process completed with exit code 1.
> ```

Run `yarn dedupe:dev` and commit the updated `yarn.lock` file

File: test/data.yml.json [hunk 1]
```diff
@@ -145,43 +145,6 @@
...
-        "info_button_color": "#0087e1",
-        "primary_button_color": "#64518a",
-        "tile_background_color": "#ffffff",
-        "text_tile_background_color": "",
-        "tile_text_color": "#3a4245",
-        "title_color": "#3a4245",
-        "warn_button_color": "#980c11",
-        "tile_title_alignment": "center"
-      }}
-  ],
```
Comment (jkaster) [inline test/data.yml.json:184]: these weren't used in any functional tests?

**Review by jkaster — APPROVED** [review]
If this passes CI, looks fine to me.

**Pull request closed without merging**